build.gradle

```diff
@@ -95,6 +98,10 @@ subprojects {
     }
 
     tasks.register('slowTest', Test) {
+        // Limit maxParallelForks to half the CPU cores or 1
+        maxParallelForks = Math.min(maxParallelForks as Integer, Runtime.runtime.availableProcessors() / 2 as Integer)
```

Comment:
what was it set to before?  Why half the cores for slow tests?  Won't each of these projects create its own forked set of runners?  I'm worried that 8 slow tests could create 8x(# cores)/2 different tests - and they might all be very big.

Reply:
max parallel forks creates a number of tasks that gradle submits to a worker pool, without increasing it via `--max-workers` command line flag, this will create `8x(# cores)/2` tasks, but only up to `# cores` could be executed at the same time.

Furthermore, unless `--parallel` is provided on the command line, we will not execute different types of tasks concurrently, so we will fan out for a specific test (up to it's allowed forks and global workers), but not execute different tasks at once (e.g. RFS:slowTest and replayer:slowTest)